homeassistant/components/automate/translations/en.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This file does not match strings.json

Reply:
I have re-run `python3 -m script.translations develop` - hopefully this will then be correctly updated. (I have never been able to get these translation strings to work - this were copied from other integrations, I always just get the full key string, not the translated text). Will be in my next push.

Reply:
Further to this, after running `python3 -m script.translations develop` a few more times after my various mis-steps, it appears `en.json` now has the translated strings (first time I have seen that happen). Not sure what happened or why, but at least it's showing the right thing in the UI now.